content/reference/user/reports/index.md

```diff
@@ -58,13 +58,20 @@ Each tab has the same structure except the summary that is available only for re
 
 This bar is a combination of buttons:
 
-- **Start / Abort**: Use this button to start a new run of the simulation, or stop the ongoing run (not available if you have a Viewer permission)
```

Comment:
You should describe the run bar above the tabs, as we describe this page from top to bottom

Reply:
done